<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lable_firstnum">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_secondnum">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Second number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_firstnum">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_secondnum">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multi_Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide_Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_result">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
